astropy/cosmology/connect.py

```diff
@@ -13,6 +25,10 @@
 __doctest_skip__ = __all__
 
 
+_CosmoT = TypeVar("_CosmoT", bound="Cosmology")  # NOTE: private b/c RTD error
+_MT = TypeVar("_MT", bound="Mapping")  # type: ignore[type-arg]
```

Comment:
We want to be clear that the comment applies to both. It also looks a bit strange that only one of the lines has an instruction for a type checker, but I'm not too worried about automated type checking yet.
```suggestion
# NOTE: private b/c RTD error
_CosmoT = TypeVar("_CosmoT", bound="Cosmology")
_MT = TypeVar("_MT", bound="Mapping")  # type: ignore[type-arg]
```